nRF9160Master/src/WiFi/WiFiHandler.c

```diff
@@ -13,21 +13,23 @@
 
 /*******************************************MACROS*********************************************************/
 #define MSG_SIZE 255
-#define WIFI_SSID_PWD       "realme GT 5G,s3qqyipp" //Change this line with SSID and password of choice
+//#define WIFI_SSID_PWD       "realme GT 5G,s3qqyipp" //Change this line with SSID and password of choice
 #define AWS_BROKER		    "a1kzdt4nun8bnh-ats.iot.ap-northeast-2.amazonaws.com"
 #define AWS_THING 		    "test_aws_iot"
 #define AWS_TOPIC 		    "test_aws_iot/testtopic"
 #define CFG_NUM 	        1
 #define CFG_NAME 	        "latlong"
 #define RETRY_COUNT         2
 
+char cWifiCredentials[80] = "Alcodex,Adx@2013";
```

Comment:
Add a comment here to mention that it is SSID and password

Reply:
will do that